src/qmeta/schema/migrate-4-to-5.sql

```diff
@@ -21,3 +21,12 @@ CREATE TABLE IF NOT EXISTS `QMessages` (
 ENGINE = InnoDB
 COMMENT = 'Table of messages generated during queries.';
 
+
+--
+-- Add the chunkCount column to QInfo.
+--  
+ALTER TABLE `QInfo` 
+  ADD `chunkCount` INT,
+  ADD `resultBytes` INT DEFAULT 0,
+  ADD `resultRows` INT DEFAULT 0;
```

Comment:
These need to be consistent with what was put into the default schema initialization file `migrate-None-to-5.sql.jinja`. The following should do the trick:
```
ALTER TABLE `QInfo`
  ADD `chunkCount` INT NOT NULL COMMENT 'number of chunks needed by the query',
  ADD `resultBytes` INT DEFAULT 0 COMMENT 'number of bytes in the result',
  ADD `resultRows` INT DEFAULT 0 COMMENT  'number of rows in the result';
```